libs/feature/search/src/lib/utils/service/fields.service.ts

```diff
@@ -15,17 +15,35 @@ import {
 })
 export class FieldsService {
   private fields = {
-    publisher: new SimpleSearchField('OrgForResource', 'asc', this.injector),
+    publisher: new SimpleSearchField(
+      'OrgForResourceObject.default',
```

Comment:
Why this change ? Do you have a multilingual catalog ?
Testing locally the `organisation/publisher` dropdown is empty, could you check that it works ?
Thanks

Reply:
This dropdown works for me locally when I test
![organisation-dropdown](https://github.com/geonetwork/geonetwork-ui/assets/36423106/6a9fddca-1081-4f65-89fd-4430d577a6dc)
The previous way was showing as blank for me, I assumed there was an issue with the ES aggregation

Reply:
There might an issue with the existing aggregation with certain catalogs; I suggest we don't include that change in this PR because it might have unexpected side effects, and open an issue instead which will allow us to investigate more. @alexarobu would you agree with that?

Reply:
@jahow f course, I can set these to what they were and amend my commit, I don't want to break anything :see_no_evil: